lib/viewport.js

```diff
@@ -0,0 +1,6 @@
+function ViewPort(height, width){
```

Comment:
Since you're not binding any methods to the prototype, you can probably just get away with a plain old JavaScript object. Especially, since you're only probably going to use this object for tests.

```js
var viewport = {
  width: 600,
  height: 800
};
```